internal/store/pod.go

```diff
@@ -334,6 +335,31 @@ func createPodContainerStatusLastTerminatedReasonFamilyGenerator() generator.Fam
 	)
 }
 
+func createPodContainerStatusLastTerminatedExitCodeFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGenerator(
+		"kube_pod_container_status_last_terminated_exitcode",
+		"Describes the exit code for the last container in terminated state.",
+		metric.Gauge,
+		"",
+		wrapPodFunc(func(p *v1.Pod) *metric.Family {
+			ms := make([]*metric.Metric, 0, len(p.Status.ContainerStatuses))
+			for _, cs := range p.Status.ContainerStatuses {
+				if cs.LastTerminationState.Terminated != nil {
+					ms = append(ms, &metric.Metric{
+						LabelKeys:   []string{"container", "reason"},
```

Comment:
Should I keep reason attached to the exit code as a tag? or remove it entirely?

Reply:
I think we should remove it because it's already present in a different metric